homeassistant/components/esphome/assist_satellite.py

```diff
@@ -180,11 +192,57 @@ async def async_set_configuration(
         # Ensure configuration is updated
         await self._update_satellite_config()
 
+    def _get_custom_wake_words(self) -> dict[str, VoiceAssistantExternalWakeWord]:
+        """Get a list of custom wake words."""
+        wake_words = {}
+        for config_path in self._wake_words_dir.glob("*.json"):
+            wake_word_id = config_path.stem
+
+            with open(config_path, encoding="utf-8") as config_file:
+                config_dict = json.load(config_file)
+                if not (model := config_dict.get("model")) or (
```

Comment:
We should force model files to have same filenames as config. So `ok_nabu.json` + `ok_nabu.tflite`. That simplifies it and we don't need a `model` key.

Reply:
Also, that way you don't need to be afraid that this will load something completely else because `..` is in `model` value.